.travis.yml

```diff
@@ -10,17 +10,20 @@ python:
 cache:
   pip: true
   directories:
-    - node_modules
+    - coalahtml/_coalahtml/node_modules
+    - coalahtml/_coalahtml/bower_components
 
 before_install:
   - sudo apt-get -y install python3-pip python-dev
   - sudo pip3 install -U pip setuptools virtualenvwrapper
 
   - npm install -g bower
+  - cd coalahtml/_coalahtml
   - bower install
+  - cd -
   - sudo pip3 install -r requirements.txt
 
-after_script:
+script:
```

Comment:
Earlier the "Default" was `npm test` because travis detected it as a nodejs package (because all th angular stuff was in root).
Now you need to run `npm test` manually in the angularjs folder